index.html

```diff
@@ -5,13 +5,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Place favicon below when found -->
-    <link rel="favicon" href="#" type="image/x-icon">
+    <link rel="icon" href="assets/sunfavicon.png"/>
     <link rel="stylesheet" href="stylesheets/styles.css"/>
     <title>Catch the Sun</title>
 </head>
 <body>
     <div class="header">
-        <p class="title">Catch the Sun</p>
+        <div class ="CatchTheSunLogo">
+        <!-- Placeholder Logo, needs styling -->
+        <img src="assets/logo2.png" alt="CatchTheSun">
+        </div>
```

Comment:
We can probably remove the div with class "CatchTheSunLogo" and replace the header div with something like:
<div class="header">
    <img src="assets/logo2.png" class="logo" alt="CatchTheSun"/>
</div>

Reply:
I agree with this suggestion as it combines what you were working on while removing the need for an extra 'div' container. 🙂